src/pykeen/pipeline.py

```diff
@@ -250,16 +250,142 @@ def title(self) -> Optional[str]:  # noqa:D401
+            r_emb_red = relation_reduction_model.fit_transform(r_emb)
+        else:
+            logger.debug('not reducing relation embeddings, already dim=2')
+            r_emb_red = r_emb
+
+        if plot_relations and plot_entities:
+            xmax = max(r_emb_red[:, 0].max(), e_emb_red[:, 0].max()) + margin
+            xmin = min(r_emb_red[:, 0].min(), e_emb_red[:, 0].min()) - margin
+            ymax = max(r_emb_red[:, 1].max(), e_emb_red[:, 1].max()) + margin
+            ymin = min(r_emb_red[:, 1].min(), e_emb_red[:, 1].min()) - margin
+        elif plot_relations:
+            xmax = r_emb_red[:, 0].max() + margin
+            xmin = r_emb_red[:, 0].min() - margin
+            ymax = r_emb_red[:, 1].max() + margin
+            ymin = r_emb_red[:, 1].min() - margin
+        elif plot_entities:
+            xmax = e_emb_red[:, 0].max() + margin
+            xmin = e_emb_red[:, 0].min() - margin
+            ymax = e_emb_red[:, 1].max() + margin
+            ymin = e_emb_red[:, 1].min() - margin
+        else:
+            raise ValueError  # not even possible
+
+        if plot_entities:
+            entity_id_to_label = self.model.triples_factory.entity_id_to_label
+            for entity_id, entity_reduced_embedding in enumerate(e_emb_red):
+                entity_label = entity_id_to_label[entity_id]
+                if entities and entity_label not in entities:
+                    continue
+                x, y = entity_reduced_embedding
+                ax.scatter(x, y, color='black')
+                ax.annotate(entity_label, (x + annotation_x_offset, y + annotation_y_offset))
+
+        if plot_relations:
+            relation_id_to_label = self.model.triples_factory.relation_id_to_label
+            for relation_id, relation_reduced_embedding in enumerate(r_emb_red):
+                relation_label = relation_id_to_label[relation_id]
+                if relations and relation_label not in relations:
+                    continue
+                x, y = relation_reduced_embedding
+                ax.arrow(0, 0, x, y, color='black')
+                ax.annotate(relation_label, (x + annotation_x_offset, y + annotation_y_offset))
+
+        if r_emb.shape[1] == 2 and e_emb.shape[1] == 2:
+            subtitle = ''
+        elif kwargs:
+            subtitle = ", ".join("=".join(item) for item in kwargs.items())
+            subtitle = f' using {model} ({subtitle})'
+        else:
+            subtitle = f' using {model}'
+        ax.set_title(f'Entity/Relation Plot{subtitle}')
+
+        if apply_limits:
+            ax.set_xlim([xmin, xmax])
+            ax.set_ylim([ymin, ymax])
+
+        return ax
+
+    def plot(self, er_kwargs: Optional[Mapping[str, str]] = None, figsize=(10, 4)):
+        """Plot all plots."""
+        import matplotlib.pyplot as plt
+        fig, (lax, rax) = plt.subplots(1, 2, figsize=figsize)
+
+        self.plot_losses(ax=lax)
+        self.plot_er(ax=rax, **(er_kwargs or {}))
```

Comment:
Plotting the ER plot by default might not be a good option, since the intuition of the plot does only hold for TransE (and maybe some of the Trans* family). Only plotting entities seems to be reasonable though since all of the models tend to give similar vectors to similarly behaving entities.

TL;DR: change `plot_er`'s default of `plot_relations` to False, or pass `plot_relations=False` by default in this call.

Reply:
Made relations plot non-default in 567eb0bf